apps/wa/wa-case-event-handler/aat.yaml

```diff
@@ -19,6 +19,8 @@ spec:
         PROBLEM_MESSAGE_SET_STATE_TO_PROCESSED_ID_LIST:
         AZURE_SERVICE_BUS_DLQ_FEATURE_TOGGLE: true
         CLEAN_UP_STARTED_DAYS_BEFORE: 30
+      spotInstances:
+        enabled: false
```

Comment:
Maybe you don't need to have this here if you've already added `false` as the default in values.yaml?

Reply:
Removed the config as suggested Ravi. :)